src/main/java/org/prebid/cache/repository/redis/RedisPropertyConfiguration.java

```diff
@@ -1,29 +1,94 @@
 package org.prebid.cache.repository.redis;
 
+import io.lettuce.core.RedisClient;
 import io.lettuce.core.RedisURI;
+import io.lettuce.core.api.StatefulRedisConnection;
+import io.lettuce.core.api.reactive.RedisStringReactiveCommands;
+import io.lettuce.core.cluster.RedisClusterClient;
+import io.lettuce.core.cluster.api.StatefulRedisClusterConnection;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.validation.constraints.NotNull;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Configuration
 @ConditionalOnProperty(prefix = "spring.redis", name = {"host"})
 @ConfigurationProperties(prefix = "spring.redis")
 public class RedisPropertyConfiguration {
+
+    private static final int DEFAULT_PORT = 6379;
     private String host;
```

Comment:
Having `host` field that might contain both hosts and ports along with `port` field that is used only in non-clustered mode is confusing. Let's stick with having only `host` property in format `host1:port1,host2:port...` (both host part and port part are mandatory) used both for clustered and non-clustered modes.